package.json

```diff
@@ -27,8 +27,11 @@
     "install:react": "npm install --prefix test/e2e/react-app",
     "prestart:react": "kill -9 $(lsof -t -i:3001 -sTCP:LISTEN) || true",
     "start:react": "npm start --prefix test/e2e/react-app & wait-on http-get://localhost:3001",
-    "pretest:e2e": "npm run install:react && node pretest.js",
-    "test:e2e": "npm run start:react && grunt test-e2e",
+    "install:angular": "npm install --prefix test/e2e/angular-app",
+    "prestart:angular": "kill -9 $(lsof -t -i:4200 -sTCP:LISTEN) || true",
```

Comment:
Would it make more sense to include the port kill on test completion?

Reply:
It's possible that they would've started something else on the port we want to use. So, I decided to kill the port *before* I use it, rather than after